collect_app/src/main/res/values/magenta_theme.xml

```diff
@@ -58,8 +60,10 @@
         <item name="android:textDirection">locale</item>
     </style>
 
-    <style name="Widget.Collect.SearchView.Magenta.Theme" parent="Widget.AppCompat.SearchView">
-        <item name="searchHintIcon">@null</item>
+    <style name="Theme.Collect.Magenta.Dialog.Alert" parent="Theme.AppCompat.Light.Dialog.Alert">
```

Comment:
I think we should avoid making any changes to Magenta here as we probably want to look at using [Shape theming](https://material.io/develop/android/theming/shape) for Material Components themes. This would also mean switching back to `MaterialAlertDialogBuilder` but I think the switch to `AlertDialog.Builder` makes our lives much easier for the moment (and is easy to unwind).

I think for me, the next step with Magenta is to use less extreme colors, switch it to being a `DayNight` theme so we get automatic light/dark switching based on the system, add shape theming (to mimic what you've done here) and then replace our light/dark themes entirely. I'll add an issue for this. Could you just remove these changes to Magenta for the moment?

Reply:
Ok it's fixed.